mule-linter-core/src/test/groovy/com/avioconsulting/mule/linter/rule/configuration/ConsecutiveLoggersCountRuleTest.groovy

```diff
@@ -29,10 +29,12 @@ class ConsecutiveLoggersCountRuleTest extends Specification{
         List<RuleViolation> violations = rule.execute(app)
 
         then:
-        violations.size() == 2
+        println violations
```

Comment:
it's needed to print here?

Reply:
Oh, I added that for my testing, will remove this print statement.